generators/cucumber/templates/Gemfile

```diff
@@ -0,0 +1,15 @@
+source 'https://rubygems.org'
```

Comment:
I guess we need to build 1 file as template with conditions instead of supporting different version of Gemfile